cg/meta/workflow/raredisease.py

```diff
@@ -274,3 +282,18 @@ def parse_analysis(self, qc_metrics_raw: list[MetricsBase], **kwargs) -> Nextflo
         return super().parse_analysis(
             qc_metrics_raw=qc_metrics_raw, qc_metrics_model=qc_metrics_model, **kwargs
         )
+
+    def get_customer_internal_mapping_csv_path(self, case: str) -> Path:
+        """Return the path to the customer internal mapping CSV file."""
+        return Path(self.get_case_path(case)) / f"{case}_customer_internal_mapping.csv"
+
+    def export_customer_internal_mapping_csv(self, case: str, output_path: Path):
+        """Export a CSV file mapping customer sample names to internal sample IDs."""
+        LOG.info(f"Exporting customer internal mapping CSV for case {case} to {output_path}")
+        with output_path.open("w", newline="") as csvfile:
+            writer = csv.writer(csvfile)
+            writer.writerow(["customer_id", "internal_id"])
+            for link in self.status_db.get_case_by_internal_id(case).links:
+                customer_name = link.sample.name
```

Comment:
I think `customer_id` and `customer_name` are a bit confusing as they are actually referring to the sample name. Could they be renamed?

Reply:
Done